docs/docs/reference/config-files/gatsby-config.md

```diff
@@ -197,6 +197,7 @@ Optionally, you can configure its behavior by passing an object to `graphqlTypeg
 module.exports = {
   graphqlTypegen: {
     typesOutputPath: `gatsby-types.d.ts`,
+    documentSearchPaths: [`./gatsby-node.ts`, `./plugins/**/gatsby-node.ts`],
```

Comment:
You'll also want to add a h3 heading called documentSearchPaths to this (like with typesOutputPath). It should explain that if you add the option, you're overwriting the defaults. So if people want to have defaults + their options they need to add the defaults themselves.